juju/names/names.go

```diff
@@ -13,4 +13,5 @@ const (
 	JujuExec        = "juju-exec"
 	JujuDumpLogs    = "juju-dumplogs"
 	JujuIntrospect  = "juju-introspect"
+	JujuDBREPL      = "juju-db-repl"
```

Comment:
Unused (atm)

I know we were thinking about using this to run the repl from a symlink to `jujud`, but this should be added when we do that